### PR TITLE
Support post-quantum MLS ciphersuite

### PR DESCRIPTION
**Pull request by pcapriotti**
This PR adds support for MLS ciphersuite `MLS_128_X25519Kyber768Draft00_AES128GCM_SHA256_Ed25519`.

https://wearezeta.atlassian.net/browse/WPB-2862

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/brig/test/integration/API/MLS.hs [hunk 1]
```diff
@@ -50,7 +50,7 @@ tests m b opts =
       test m "GET /mls/key-packages/self/:client/count" (testKeyPackageZeroCount b),
       -- FUTUREWORK test m "GET /mls/key-packages/self/:client/count (expired package)" (testKeyPackageExpired b),
       test m "GET /mls/key-packages/claim/local/:user" (testKeyPackageClaim b),
-      test m "GET /mls/key-packages/claim/local/:user - self claim" (testKeyPackageSelfClaim b),
+      test m "XXX" (testKeyPackageSelfClaim b),
```
Comment (smatting) [inline services/brig/test/integration/API/MLS.hs:53]: Leftover?

File: services/galley/src/Galley/Intra/Client.hs [hunk 2]
```diff
@@ -182,7 +182,12 @@ getLocalMLSClients lusr ss =
             "clients",
             toByteString' (tUnqualified lusr)
           ]
-        . queryItem "sig_scheme" (toByteString' (signatureSchemeName ss))
+        . queryItem
+          "ciphersuite"
+          ( "0x"
+              <> toByteString'
+                (Hex (cipherSuiteNumber (tagCipherSuite suite)))
```
Comment (smatting) [inline services/galley/src/Galley/Intra/Client.hs:189]: Use `instance ToHttpApiData CipherSuite` for consistency

Comment (pcapriotti) [reply services/galley/src/Galley/Intra/Client.hs]: Fixed

File: services/brig/test/integration/API/MLS.hs (hunk 1 above)
Comment (pcapriotti) [reply services/brig/test/integration/API/MLS.hs]: Fixed

**Pull request merged**